### PR TITLE
Set target temperature

### PR DESCRIPTION
**Pull request by filipgoranson**
Allows the configuration to add a target temperature source.

### CODE REVIEW

**Comment by elupus** [comment]
Why? Is the default setpoint value wrong?

**Comment by elupus** [comment]
The target for a smart home entity is the climate entity. For a pump it's the offset, for a climate with temp sensor it's the real target.

Please explain what you are trying to solve.

**Comment by filipgoranson** [comment]
I have room thermostats where i enter the setpoint. They give the setpoint and current temperature to HA. I want the setpoint from these to be the value used by nibe. Is there a better way to connect the target temperature?

**Comment by elupus** [comment]
Link current temperature to the smart termostat 's current temperature, then add an automation that update the smart home termostat when your external control changes.

**Comment by elupus** [comment]
But that said, I would mind supporting an external entity as set point. But this change is modifying other things.

**Comment by elupus** [comment]
My main concern is the set_temperature change.

File: climate.py [hunk 1]
```diff
@@ -225,6 +227,7 @@ def __init__(
     ):
         """Init."""
         super().__init__(uplink, system_id, statuses, climate)
+        self._target_id = None
```
Comment (elupus) [inline climate.py:230]: I can't see that you ever set this value?

**Comment by elupus** [comment]
Lack of update from author

**Pull request closed without merging**